packages/Checkboxes/__tests__/Checkbox.test.tsx

```diff
@@ -40,67 +42,53 @@ const defaultProps = (override = {}) => ({
   ...override,
 });
 
-describe("Checkboxes component", () => {
-  test("should be defined and renders correctly (snapshot)", () => {
-    const wrapper = shallow(<Checkboxes {...defaultProps()} />);
-
-    expect(wrapper).toBeDefined();
-    expect(wrapper).toMatchSnapshot();
+describe('Checkboxes component', () => {
+  it('should be defined and renders correctly (snapshot)', () => {
+    const { asFragment } = render(<Checkboxes {...defaultProps()} />);
+    expect(asFragment()).toMatchSnapshot();
   });
 
-  test("should render and toggle single checkbox", () => {
+  it('should render and toggle single checkbox', () => {
```

Comment:
This isn't toggling a checkbox anymore it should probably be firing a click event and checking it is checked or change test description to something like `should render single checkbox`

Reply:
`fixme`